magento/magento2
# Fix #24710 - Update countries list

### Description (*)

This is fix for #24710
Note, that Canary Islands weren't added as it is part of Spain and doesn't contain it's own iso_3 code

### Fixed Issues (if relevant)

1. magento/magento2#24710: Update countries list / CLDR to version 36

### Manual testing scenarios (*)

1. Go to Stores -> Configuration -> General -> General -> Country Options
2. Check if **Allow Countries** contain following countries: Caribbean Netherlands, Curacao, Kosovo, Sint Maarten.

### Questions or comments


### Contribution checklist (*)
